js/filter_type.js

```diff
@@ -31,9 +31,9 @@ $(document).ready(function(){
                     var result = [];
 
                     for (var l = 0; l < sortOpt.length; l++){
-                        filtersStr += "<label class='col-md-4 control-label'>Filter" + (l + 1) + "&nbsp;</label>";
+                        filtersStr += "<label class='col-md-4 control-label'>Filter " + (l + 1) + "&nbsp;</label>";
                         filtersStr += "<select id='filter" + (l + 1) + "' class='form-control selectpicker'>";
-                        filtersStr += "<option selected value='justNull'> - " + sortOpt[l] + " - </option>";
+                        filtersStr += "<option selected value='null'> - " + sortOpt[l] + " - </option>";
```

Comment:
There's no semantical difference between `"justNull"` and `"null"`, they are both made up strings. You could have called it `SLAC-Gismo`, as long as that's what you ended up comparing it to later.

My suggestion was to use something semantically different, like actually `null || undefined`. Or even something not semantically too different but that evaluates to false when coerced to a bool. e.g.: `""` vs. `"some-string"`

Reply:
Fixed